recipe/meta.yaml

```
@@ -12,28 +14,25 @@ source:
     - setup.py.patch
 
 build:
-  number: 1
+  number: 2
   skip: true  # [not linux]
-  skip: true  # [cuda_compiler_version == "None"]
-  skip: true  # [cuda_compiler_version == "10.2"]
-  skip: true  # [cuda_compiler_version == "11.0"]
+  skip: true  # [cuda_compiler_version != "11.6"]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-    - setuptools
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
   host:
     - python
     - pip
-    - setuptools
+    - cudatoolkit {{ cuda_version }}.*
```

Can just use this with the CUDA compiler used above

```suggestion
    - cudatoolkit {{ cuda_compiler_version }}
```